src/language/lang_id.php

```diff
@@ -0,0 +1,659 @@
+<?hh // strict
+
+/* HH_IGNORE_ERROR[1002] */
+$translations = array(
+  'date and time format' =>
+    'D, d-m-Y H:i:s', //used by date() function
+  //Translations for IndexController
+  'Facebook CTF' =>
+    'Facebook CTF',
+  'Conquer the world' =>
+    'Kuasai dunia',
+  'Play' =>
+    'Mulai',
+  'Welcome to the Facebook Capture the Flag Competition. By clicking "Play," you will be entered into the official CTF challenge. Good luck in your conquest.' =>
+    'Selamat datang di Kompetisi Facebook Capture the Flag. Dengan menekan "Mulai" maka kamu akan dapat mulai bermain di Kompetisi Facebook CTF. Semoga beruntung dalam menguasai dunia.',
```

Comment:
Suggested change: 'Selamat datang di Kompetisi Facebook Capture the Flag. Dengan menekan "Main" maka kamu akan dapat mulai bermain di Kompetisi Facebook CTF. Semoga beruntung dalam menguasai dunia.'

Comment: Mistranslation of the term "Play" into "Start"